src/Codeception/Util/Maybe.php

```diff
@@ -39,7 +38,7 @@ class Maybe implements ArrayAccess, Iterator, JsonSerializable
      */
     protected $position = 0;
     /**
-     * @var null|object
+     * @var ArrayAccess|array|object|null
```

Comment:
How can it be object if constructor permits only array and null?

Reply:
Well, in various places in the file `is_object` and `property_exists` are used with this property (`$val`), maybe that typing is wrong too (?).

Reply:
I think that it would be better to set it to `mixed` and remove all typehints.

There wree no restrictions on value types before.